files/mark.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "mark.js",
+  "repo": "julmot/mark.js",
+  "files": {
+    "basePath": "dist/"

```

Comment:
jsDelivr is focused on production usage. But if you need the unminified files then we can leave them.